common/models/user.js

```diff
@@ -577,6 +578,15 @@ module.exports = function(User) {
         return cb(err);
       }
 
+      var AccessToken = UserModel.relations.accessTokens.modelTo;
+      AccessToken.find({ where: { userId: user.id }}, function(err, tokens) {
+        tokens.forEach(function(token) {
+          AccessToken.destroyById(token.id, function(err, info) {
+            if (err) return cb(err);
```

Comment:
This will can potentially call `cb` multiple times, once for each time an error happens.